### DISCUSSION TITLE
Support stand-alone vector database

### DISCUSSION THREAD

**Discussion by DAYUCS** [open-webui/open-webui#5427] [General]
The built-in RAG in Open WebUI is very impressive and very convenient to use for building RAGs. However, the performance drops significantly when the number of documents increases. Please enhance this feature to also use a stand-alone vector database, which will definitely further expand its scope of application.

**Comment by gravity-api** [comment]
This is already supported.
https://docs.openwebui.com/getting-started/env-configuration/#rag
https://github.com/open-webui/open-webui/discussions/938

```none
environment:
      - 'CHROMA_HTTP_HOST=ip/hostname'
      - 'CHROMA_HTTP_PORT=8000'
```
